boards/arm/nrf5340dk_nrf5340/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 # Copyright (c) 2019 Nordic Semiconductor ASA.
 # SPDX-License-Identifier: Apache-2.0
 
-if ((CONFIG_BOARD_NRF5340PDK_NRF5340_CPUAPP OR CONFIG_BOARD_NRF5340PDK_NRF5340_CPUAPPNS)
+if (((CONFIG_BOARD_NRF5340PDK_NRF5340_CPUAPP OR CONFIG_BOARD_NRF5340PDK_NRF5340_CPUAPPNS)
+    OR (CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPP OR CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPPNS))
```

Comment:
I am wondering if this bunch of CONFIG options could be replaced by `CONFIG_SOC_SERIES_NRF53X` ?
In fact each nrf5340(p)dk_nrf5340_cpuapp_defconfig enables `CONFIG_SOC_SERIES_NRF53X=y` so it could be simpler and cleaner. I think including this source is SoC specific not board specific. Could be similar for all over the place. What do You think?

Reply:
I agree, there has to be a better solution than this. If not the series, then maybe a new Kconfig option named `CONFIG_BOARD_NRF5340XDK` that is selected by both the PDK and DK boards.

Reply:
@carlescufi @emob-nordic what you are asking cannot be done, since either of the proposals, CONFIG_SOC_SERIES_NRF53X, or CONFIG_BOARD_NRF5340XDK would also be set when we build for the Network MCU.

The only way to clean this up would be to introduce APP/NET -specific, but DK-generic Kconfig options :) I was thinking this is not worth the effort, since we will probably not going to keep the PDK support in the Project, for too long. Eventually we will have only the "DK" support, and any common options we introduce will be very short-lived and could lead to more confusion. So I didn't want to do it, unless you insist.

Reply:
I don't have strong opinion here but I thought it would be nice to shorten those `ifdefs` as they look almost the same and reader could easily misunderstand this.
If we plan to remove PDK support soon then Your argumentation seems logic for me and I would opt for it.
If we plan **not** to remove PDK support then I think introducing DK-genric Kconfig option is worthwhile.

PS: Do we really want to delete PDK support in the near future? I though its like "once in, never out".

Reply:
I think we will delete it as soon as we broadly introduce the final DKs